sky/cli.py

```diff
@@ -1493,7 +1493,9 @@ def cost_report(all: bool):  # pylint: disable=redefined-builtin
         if cluster_name in backend_utils.SKY_RESERVED_CLUSTER_NAMES:
             cluster_group_name = backend_utils.SKY_RESERVED_CLUSTER_NAMES[
                 cluster_name]
-            reserved_clusters[cluster_group_name] = cluster_record
+            reserved_clusters[
+                cluster_group_name] = status_utils.nullify_cost_for_autostop(
+                    cluster_record)
```

Comment:
Autostop clusters are not necessarily be `reserved_clusters`. Please test the PR with the following:
```
sky launch -c min -i 0 --cloud
sky status
sky cost-report
```